dev-docs/bidders/spotx.md

```diff
@@ -12,24 +12,39 @@ biddercode: spotx
+| `outstream_options`  | optional | Object to set options on the renderer. See [outstream_options Object](#spotx-outstream-options-object)                                                              | `{}`                                                                                                                                                                                                 | `object`    |
+| `secure`             | optional | Boolean identifying whether the reqeusts should be https or not (used to override the protocol if the page isn't secure.                                      | `true`                                                                                                                                                                                               | `boolean`   |
+| `mimes`              | optional | List of mimetypes to allow in ad.                                                                                                                             | `['application/javascript', 'video/mp4', 'video/webm']                                                                                                                                               | `array`     |
+| `ad_volume`          | optional | Value between 0 and 1 to denote the volume the ad should start at                                                                                             |                                                                                                                                                                                                      | `float`     |
+| `price_floor`        | optional | Set the current channel price floor in real time.                                                                                                             | `10`                                                                                                                                                                                                   | `integer`   |
+| `hide_skin`          | optional | Set to true to hide the spotx skin                                                                                                                            |                                                                                                                                                                                                      | `boolean`   |
+| `custom`             | optional | See SpotX documentation [here](https://developer.spotxchange.com/content/local/docs/sdkDocs/DirectSdk/README.md#custom-property-for-key-value-pair-reporting) |                                                                                                                                                                                                      | `object`    |
+
+<a name="spotx-outstream-options-object
```

Comment:
Link object seems to be broken

Reply:
When I ran this through a markdown parser, the link worked as expected. This github repo isn't parsing these right, but it should look correct when it's published.

Reply:
awesome. Thanks :heart: